CHANGELOG.md

```
@@ -1,3 +1,66 @@
+# 3.5.0
+> Published 14 May 2026
+
+### Features
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)